test/test.js

```diff
@@ -0,0 +1,232 @@
+mocha.ui('bdd');
+mocha.reporter('html');
+
+describe('hermes', function () {
+    'use strict';
+
+    var assert = chai.assert;
+
+    describe('setup', function () {
+        it('should attach a list to the provided element', function () {
```

Comment:
I think we have some rule about not using "should" in tests, since it's not very valuable